### DISCUSSION TITLE
Any way I can keep bumping the version of openvscode in our dockerfiles using dependabot?

### DISCUSSION THREAD

**Discussion by bhavitsharma** [gitpod-io/openvscode-server#385] [General]
Any way I can keep bumping the version of openvscode in our dockerfiles using dependabot?
Currently, I have to bump the version in our Dockerfile

**Comment by filiptronicek** [comment]
Hey there @bhavitsharma 👋! 

I think [using Dependabot with Docker](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem=) and using a tagged version of OpenVSCode Server in your Dockerfile should do the trick. If it doesn't, you could build a simple GitHub action to change the Dockerfile to the latest version. You can get this latest version by using a command like this:

```bash
curl https://api.github.com/repos/gitpod-io/openvscode-server/releases | jq ".[0].tag_name"
```
